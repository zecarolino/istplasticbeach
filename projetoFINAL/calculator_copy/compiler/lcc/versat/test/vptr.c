int f() { return 1; }
int g() { return 2; }
int h() { return 3; }
int (*v[])() = { f, g, h };
int main() { return v[2](); }
