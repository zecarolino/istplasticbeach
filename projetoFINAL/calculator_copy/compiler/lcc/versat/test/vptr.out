VCD info: dumpfile xtop.vcd opened for output.
3
Catched Trap at time  420 clock cycles:
